sdk/storage/Azure.Storage.Blobs/AzureStorageNetMigrationV12.md

```diff
@@ -66,7 +66,7 @@ v12
 A `TokenCredential` abstract class (different API surface than v11) exists in the Azure.Core package that all libraries of the new Azure SDK family depend on, and can be used to construct Storage clients. Implementations of this class can be found separately in the [Azure.Identity](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity) package. [`DefaultAzureCredential`](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#defaultazurecredential) is a good starting point, with code as simple as the following:
 
 ```C# Snippet:SampleSnippetsBlobMigration_TokenCredential
-BlobServiceClient client = new BlobServiceClient(new Uri(accountUri), new DefaultAzureCredential());
+BlobServiceClient client = new BlobServiceClient(new Uri(serviceUri), new DefaultAzureCredential());
```

Comment:
should this be changed in the `C# Snippet:SampleSnippetsBlobMigration_TokenCredential`?

Reply:
yes it should